gammapy/stats/variability.py

```diff
@@ -228,3 +229,65 @@ def compute_flux_doubling(flux, flux_err, coords, axis=0):
     }
 
     return doubling_dict
+
+
+def TimmerAlgorithm(power_spectrum, even_freq=True):
+    """Implementation of the Timmer algorithm to simulate a time series from a power spectrum
```

Comment:
Also, take `random_state` as input to configure the random number generator. See `MapDataset.fake` for ex

Reply:
Thanks, this is a very important suggestion.